p2p/src/main/java/bisq/network/p2p/storage/persistence/StoreService.java

```diff
@@ -139,9 +139,9 @@ protected void readStore() {
         final String fileName = getFileName();
         store = storage.initAndGetPersistedWithFileName(fileName, 100);
         if (store != null) {
-            log.info("{}: size of {}: {} kb", this.getClass().getSimpleName(),
+            log.info("{}: size of {}: {} MB", this.getClass().getSimpleName(),
                     storage.getClass().getSimpleName(),
-                    store.toProtoMessage().toByteArray().length / 100D);
+                    store.toProtoMessage().toByteArray().length / 1_000_000D);
```

Comment:
This look correct, but it looks like the previous amount in kb was off by a factor 10.

Reply:
Yes, that was wrong ;-)